app/components/ActionLinks/tests/index.test.js

```diff
@@ -6,9 +6,9 @@ import ActionLinks from '../index'
 
 describe('Actions component', () => {
   it('should only show the my allocations link when the user is a key worker', () => {
-    const wrapper = shallow(<ActionLinks isKeyWorker />)
+    const wrapper = shallow(<ActionLinks omicUrl="//omicURL" staffId="123" isKeyWorker />)
```

Comment:
Why the double forward slash?

Reply:
Oh it was just copying the other test below, can't really claim I thought about it at all :D